Functions-request-config.js

```diff
@@ -1,46 +1,28 @@
 const fs = require("fs")
+const { Location, ReturnType, CodeLanguage } = require("@chainlink/functions-toolkit")
 
 // Loads environment variables from .env.enc file (if it exists)
 require("@chainlink/env-enc").config()
 
-const Location = {
-  Inline: 0,
-  Remote: 1,
-}
-
-const CodeLanguage = {
-  JavaScript: 0,
-}
-
-const ReturnType = {
-  uint: "uint256",
-  uint256: "uint256",
-  int: "int256",
-  int256: "int256",
-  string: "string",
-  bytes: "Buffer",
-  Buffer: "Buffer",
-}
-
 // Configure the request by setting the fields below
 const requestConfig = {
   // Location of source code (only Inline is currently supported)
   codeLocation: Location.Inline,
+  // Location of secrets (only Remote or DONHosted is supported)
+  secretsLocation: Location.DONHosted,
   // Code language (only JavaScript is currently supported)
   codeLanguage: CodeLanguage.JavaScript,
   // String containing the source code to be executed
   source: fs.readFileSync("./calculation-example.js").toString(),
   //source: fs.readFileSync('./API-request-example.js').toString(),
   // Secrets can be accessed within the source code with `secrets.varName` (ie: secrets.apiKey). The secrets object can only contain string values.
-  //secrets: { apiKey: process.env.COINMARKETCAP_API_KEY ?? "" },
-  // Per-node secrets objects assigned to each DON member. When using per-node secrets, nodes can only use secrets which they have been assigned.
-  perNodeSecrets: [],
+  secrets: { apiKey: process.env.COINMARKETCAP_API_KEY ?? "" },
   // ETH wallet key used to sign secrets so they cannot be accessed by a 3rd party
   walletPrivateKey: process.env["PRIVATE_KEY"],
   // Args (string only array) can be accessed within the source code with `args[index]` (ie: args[0]).
   args: ["1", "bitcoin", "btc-bitcoin", "btc", "1000000", "450"],
   // Expected type of the returned value
-  expectedReturnType: ReturnType.uint256,
+  expectedReturnType: ReturnType.string,
   // Redundant URLs which point to encrypted off-chain secrets
   secretsURLs: [],
```

Comment:
worth clarifying comment with
// Redundant (un-encryped) URLs which point to encrypted off-chain secrets. Encryption will be done by functions-toolkit

Many users will wonder if they need to encrypt the URLs before putting them in this array.

Reply:
They are actually no longer used as we just default to using the automatically created Gist URLs in `request.js` (which I think is way less confusing for beginners).  I removed secretsURLs to avoid confusion.